### PR TITLE
Feature/animated phone block

### PR DESCRIPTION
**Pull request by isaacSennerholt**

### CODE REVIEW

File: src/blocks/AnimatedPhoneBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+} from '../components/blockHelpers'
+
+import { getStoryblokLinkUrl } from '../utils/storyblok'
+
+const ButtonLinkWithMargin = styled(ButtonLink)({
+  marginTop: '1.7rem',
+})
+
+const AlignableContentWrapper = styled(ContentWrapper)(
+  ({ textPosition }: { textPosition: string }) => ({
```
Comment (palmfjord) [inline src/blocks/AnimatedPhoneBlock.tsx]: `TextPosition` no?

File: src/blocks/AnimatedPhoneBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    flexDirection: textPosition === 'right' ? 'row-reverse' : 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    [TABLET_BP_DOWN]: {
+      flexDirection: 'column',
+    },
+  }),
+)
+
+const TextWrapper = styled('div')(
```
Comment (palmfjord) [inline src/blocks/AnimatedPhoneBlock.tsx]: same, `string` -> `TextPosition`?

File: src/blocks/AnimatedPhoneBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    <SectionWrapper color={color && color.color} size={size}>
+      <AlignableContentWrapper textPosition={text_position}>
+        <TextWrapper textPosition={text_position}>
+          <Title size={size} displayOrder={media_position}>
+            {title}
+          </Title>
+          <Paragraph
+            size={size}
+            dangerouslySetInnerHTML={{
+              __html: paragraph.html,
```
Comment (palmfjord) [inline src/blocks/AnimatedPhoneBlock.tsx]: i think we should do `paragraph && paragraph.html` just to be safe here

**Comment by palmfjord** [comment]
Also @isaacSennerholt feel free to assign everyone in @HedvigInsurance/clients to review the PR :)

File: package.json [hunk 4]
```diff
@@ -36,8 +36,10 @@
     "@types/react-dom": "^16.0.9",
     "@types/react-helmet-async": "^0.0.1",
     "@types/react-hot-loader": "^4.1.0",
+    "@types/react-lottie": "^1.2.0",
```
Comment (palmfjord) [inline package.json]: do we need this now that we have our own lottie types?

File: src/blocks/AnimatedPhoneBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    '&:>div': {
+      width: '100%',
+    },
+    display: alignment === 'center' ? 'none' : 'block',
+    [TABLET_BP_DOWN]: {
+      maxWidth: '100%',
+      width: 'auto',
+      marginTop: displayOrder === 'top' ? '0' : '3rem',
+      display: 'block',
+      order: displayOrder === 'top' ? -1 : 'initial',
```
Comment (palmfjord) [inline src/blocks/AnimatedPhoneBlock.tsx]: imo intitial has a little too poor browser compatibility :/ https://caniuse.com/#feat=css-initial-value could we just do `undefined` instead or does it not work here?

File: src/components/LazyLottie.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      {({
+        animationData,
+        setLoadedAnimationData,
+        ReactLottie,
+        setLoadedLottie,
+      }: State & Actions) => (
+        <Mount
+          on={() => {
+            props.options.animationData.then(setLoadedAnimationData)
+            import('react-lottie').then((lottieModule) => {
```
Comment (palmfjord) [inline src/components/LazyLottie.tsx]: maybe we could add a chunk name here? like so https://github.com/webpack/webpack/tree/master/examples/code-splitting-specify-chunk-name

File: src/blocks/AnimatedPhoneBlock.tsx [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+              bold
+            >
+              {button_title}
+            </ButtonLinkWithMargin>
+          )}
+        </TextWrapper>
+        <PhoneContainer alignment={text_position} displayOrder={media_position}>
+          <LazyLottie
+            options={{
+              animationData: import('animations/chatDemo.json'),
```
Comment (palmfjord) [inline src/blocks/AnimatedPhoneBlock.tsx]: and same here, could we add a chunk name? https://github.com/webpack/webpack/tree/master/examples/code-splitting-specify-chunk-name

File: package.json (hunk 4 above)
Comment (isaacSennerholt) [reply package.json]: Good catch!

**Pull request merged**